### ISSUE TITLE
Area counter not reset

### ISSUE THREAD

**Issue by vviitani** [farmerbriantee/AgOpenGPS#42]
When using the "Delete Applied" button to remove coverage map for a field, the area counter below the auto sections button is not reset. I think it would be logical to reset it so the value would represent the real applied area.

**Comment by vviitani** [comment]
I mean this counter in the pic. It starts from zero when creating a new field, and the value is stored and recalled with each field. 

Now that I played more with it, I noticed that pressing the "Delete Applied" doesn't reset the shown counter value, but after closing and re-opening the field the counter is in fact reset as expected.

![ha_counter](https://user-images.githubusercontent.com/39761670/41199458-3d079ec6-6c9b-11e8-93e5-37b235891be9.jpg)
